ZuluContent/Items/Addons/BaseAddon.cs

```diff
@@ -100,28 +80,24 @@ public virtual void OnChop(Mobile from)
 
                 if (deed != null)
                 {
-                    if (RetainDeedHue)
-                        deed.Hue = hue;
+                    if (deed is BaseAddonDeed addonDeed)
```

Comment:
Don't need the above null check if you're doing a pattern match here, it serves the same purpose of ensuring the variable is of a type and not null.